hack/spectest-report.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -x
+
+# Prysm spectest directory to start from
+start_directory="/prysmaticLabs/prysm/testing/spectest"
+# Can be found with os.Getenv("TEST_UNDECLARED_OUTPUTS_DIR")
+bazel_output_dir="<path to bazel>/testlogs/testing/spectest"
+
+# URL of the Ethereum Spec Tests GitHub repo
+github_repo_url="git@github.com:ethereum/consensus-spec-tests.git"
+# Temporary directory to clone the GitHub repository
+temp_dir=$(mktemp -d)
+
+# Clone the GitHub repository
+git clone "$github_repo_url" "$temp_dir"
```

Comment:
This is a bit expensive to download on every run. Could you download it once and reuse the repo on subsequent runs?